prereise/gather/griddata/hifld/data_process/transmission.py

```diff
@@ -559,12 +672,8 @@ def _euclidian(a, b):
         )
         return const.transformer_reactance[closest_voltage_tuple]
     elif branch.loc["type"] == "Line":
```

Comment:
This is outside this PR: given branch is a series, I believe we don't need `loc` here. Same comment below.

Reply:
That's true. This is taken care of in #286, so it should become a moot point soon.